quipucords/quipucords/settings.py

```diff
@@ -321,6 +321,9 @@ def app_secret_key_and_path():
 
 STATICFILES_DIRS = [BASE_DIR / "client"]
 
+if not PRODUCTION:
+    TESTDATA_DIR = BASE_DIR / "testdata"
```

Comment:
Surprise! 🎉 What do you think about this addition? If this is okay, then I hope we could expand use of this new `TESTDATA_DIR` for future unit tests that might benefit from large sets of static data, possibly captured from real-word commands like I did here for hostnamectl.

Reply:
my first thought is this seems to be reinventing the wheel (see [tmp_path fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture)), but I just started looking at the PR.

Reply:
ah, I see now what its purpose is. seems good to me =D